dts/bindings/spi/silabs,gecko-spi-eusart.yaml

```diff
@@ -0,0 +1,31 @@
+description: GECKO EUSART SPI
+
+compatible: "silabs,gecko-spi-eusart"
+
+include: [spi-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  peripheral-id:
+    type: int
+    description: peripheral ID
+
+  # Note: Not all SoC series support setting individual pin location. If this
```

Comment:
The rest of this file from this point can be removed. The `location-*` properties should no longer be used, pinctrl should be used instead. So this new binding doesn't need to define them. Even if the properties were kept, this comment is inaccurate since EUSART only exists on Series 2, which uses DBUS for pin configuration, and always supports individual pin locations.

Reply:
Done, I removed the part of the code you suggested.

Thanks for your feedback!